packages/web3torrent/src/constants.ts

```diff
@@ -183,7 +184,7 @@ export const INITIAL_BUDGET_AMOUNT = utils.hexZeroPad(
 
 export const MAX_FILE_SIZE = 1000 * 1024 * 1024;
 
-export const LOG_DESTINATION = process.env.LOG_DESTINATION;
+export const LOG_DESTINATION = path.join(process.env.LOG_DESTINATION, 'web3torrent.log');
```

Comment:
Do we need to check if `process.env.LOG_DESTINATION` is null or an empty string?

Reply:
We do ...